README.md

```diff
@@ -344,146 +344,124 @@ to get started setting up your self-hosted runner.
 ### Allocating cloud resources with CML
 
 When a workflow requires computational resources (such as GPUs) CML can
-automatically allocate cloud instances. For example, the following workflow
+automatically allocate cloud instances using `cml-runner`. You can spin up instances on your AWS or Azure account (GCP support is forthcoming!). 
+
+For example, the following workflow
 deploys a `t2.micro` instance on AWS EC2 and trains a model on the instance.
-After the instance is idle for 120 seconds, it automatically shuts down.
+After the job runs, the instance automatically shuts down.
 
 ```yaml
-name: train-my-model
+name: "Train-in-the-cloud"
 on: [push]
+
 jobs:
-  deploy-cloud-runner:
+  deploy-runner:
     runs-on: [ubuntu-latest]
-    container: docker://dvcorg/cml:latest
     steps:
+      - uses: hashicorp/setup-terraform@v1
+      - uses: iterative/setup-cml@v1
```

Comment:
This might be too much detail, but since you mention the `hashicorp/setup-terraform@v1` dependency below, what is `iterative/setup-cml@v1` for?

Reply:
This is actually going to be removed because `iterative/setup-cml@v1` has absorbed the `setup-terraform` dependency- but originally, `setup-terraform` was not sufficient to run `cml-runner`. That function comes with the CML library (I'm only 80% sure this is what you were asking, so please clarify if this isn't what you meant)